source/fuzz/transformation_load.h

```diff
@@ -28,11 +28,20 @@ class TransformationLoad : public Transformation {
   explicit TransformationLoad(protobufs::TransformationLoad message);
 
   TransformationLoad(
-      uint32_t fresh_id, uint32_t pointer_id,
+      uint32_t fresh_id, uint32_t pointer_id, bool is_atomic,
+      uint32_t memory_scope, uint32_t memory_semantics,
       const protobufs::InstructionDescriptor& instruction_to_insert_before);
 
   // - |message_.fresh_id| must be fresh
   // - |message_.pointer_id| must be the id of a pointer
+  // - |message_.is_atomic| must be true if want to work with OpAtomicLoad
+  // - |message_.memory_scope_id| if |is_atomic| is true then this must be the
```

Comment:
Rephrase to: "- If |is_atomic| is true then |message_memory_scope_id| must be the ..."

Also, make sure your comments are indented so that when a bullet point spans multiple lines, the additional lines are aligned with the start of the bullet text.

On the line below, `id of an ...` should align with `|message_....` on the line above, for example.

Reply:
Done.